sandbox/apparmor/apparmor.go

```diff
@@ -486,6 +493,9 @@ func AppArmorParser() (cmd *exec.Cmd, internal bool, err error) {
 	for _, dir := range filepath.SplitList(parserSearchPath) {
 		path := filepath.Join(dir, "apparmor_parser")
 		if _, err := os.Stat(path); err == nil {
+			if fi, err := os.Lstat(hostAbi30File); err == nil && !fi.IsDir() {
```

Comment:
There is a second reference to the 3.0 ABI in this function, notably line 486 - could we incorporate this as well so we don't end up having it defined twice?

Reply:
Those are really different, the one on line 486 is relative to the snapd snap.